sql/moz-fx-data-shared-prod/stripe_external/customers_changelog_v1/metadata.yaml

```diff
@@ -0,0 +1,27 @@
+friendly_name: Stripe customers changelog
+description: |-
+  Changes to Stripe customers as recorded by Fivetran (https://fivetran.com/docs/applications/stripe).
+
+  IMPORTANT: This table should not be rebuilt from scratch because the underlying Fivetran table doesn't preserve historical changes.
+
+  This table's schema closely mirrors Stripe's customers API (https://stripe.com/docs/api/customers/object).
+  Fields which Fivetran doesn't sync and fields we've specifically chosen not to sync (e.g. PII) have been omitted.
+owners:
+- srose@mozilla.com
+labels:
+  incremental: true
+  schedule: daily
+scheduling:
+  dag_name: bqetl_subplat
+  # The table as a whole is appended to each time, not a specific date partition.
+  date_partition_parameter: null
+  arguments:
+  - --append_table
```

Comment:
I chose this approach because only syncing the changes from the previous calendar day would miss changes that occurred after midnight if the customer was updated again before the next sync (and in the worst edge case, a customer that was repeatedly updated between midnight and when the Fivetran syncs run would never have any changes saved in the changelog).